<launch>

  <node pkg="elavation_mapping_test" type="fake_pos.py" name="fake_pos" output="screen"/>
  
  <!-- Elevation mapping node -->
  <node pkg="elevation_mapping" type="elevation_mapping" name="elevation_mapping" output="screen">
    <rosparam command="load" file="$(find elavation_mapping_test)/config/robot.yaml" />
    <rosparam command="load" file="$(find elavation_mapping_test)/config/map.yaml" />
    <rosparam command="load" file="$(find elavation_mapping_test)/config/sensor.yaml" />
    <rosparam command="load" file="$(find elavation_mapping_test)/config/postprocessor_pipeline.yaml" />
  </node>

  <!-- Launch visualizations for the resulting elevation map -->
  <include file="$(find elevation_mapping_demos)/launch/visualization.launch" />


  <!-- Launch RViz with the demo configuration -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find elevation_mapping_demos)/rviz/elevation_map_visualization.rviz" />

  <node pkg="tf" type="static_transform_publisher" name="map_to_exca"
    args="0.98 0.0 0.0 3.14159 0.0 0.0 /map /base_footprint 40" />
  


</launch>